tests/Unit/Evolution/DiscontinuousGalerkin/Test_LiftFromBoundary.cpp

```diff
@@ -0,0 +1,259 @@
+        interface_logical_coordinates(face_mesh, direction);
+    const auto face_inertial_coords = map(face_logical_coords);
+    const auto face_det_jacobian =
+        determinant(map.jacobian(face_logical_coords));
+    const auto face_normal =
+        unnormalized_face_normal(face_mesh, map, direction);
+    const auto face_normal_magnitude = magnitude(face_normal);
+    auto unit_face_normal = face_normal;
+    for (auto& t : unit_face_normal) {
+      t /= get(face_normal_magnitude);
+    }
+
+    const auto boundary_corrections = normal_dot_flux<tags>(
+        unit_face_normal,
+        polynomial_volume_fluxes<tags>(face_inertial_coords, powers));
+
+    // Now perform lifting
+    evolution::dg::lift_boundary_terms_gauss_points(
+        make_not_null(&dt_vars_lifted_one_at_a_time), volume_det_inv_jacobian,
+        volume_mesh, direction, boundary_corrections, face_normal_magnitude,
+        face_det_jacobian);
+
+    if (direction.side() == Side::Upper) {
+      // Test lifting both upper and lower correction at the same time
+      const auto lower_direction = direction.opposite();
+      const auto lower_face_mesh =
+          volume_mesh.slice_away(lower_direction.dimension());
+      const auto lower_face_logical_coords =
+          interface_logical_coordinates(lower_face_mesh, lower_direction);
+      const auto lower_face_inertial_coords = map(lower_face_logical_coords);
+      const auto lower_face_det_jacobian =
+          determinant(map.jacobian(lower_face_logical_coords));
+      const auto lower_face_normal =
+          unnormalized_face_normal(lower_face_mesh, map, lower_direction);
+      const auto lower_face_normal_magnitude = magnitude(lower_face_normal);
+      auto lower_unit_face_normal = lower_face_normal;
+      for (auto& t : lower_unit_face_normal) {
+        t /= get(lower_face_normal_magnitude);
+      }
+
+      const auto lower_boundary_corrections = normal_dot_flux<tags>(
+          lower_unit_face_normal,
+          polynomial_volume_fluxes<tags>(lower_face_inertial_coords, powers));
+
+      // Now perform lifting
+      evolution::dg::lift_boundary_terms_gauss_points(
+          make_not_null(&dt_vars_lifted_two_at_a_time), volume_det_inv_jacobian,
+          volume_mesh, direction.dimension(), boundary_corrections,
+          face_normal_magnitude, face_det_jacobian, lower_boundary_corrections,
+          lower_face_normal_magnitude, lower_face_det_jacobian);
+    }
+  }
+
+  Variables<db::wrap_tags_in<
+      Tags::div, typename std::decay_t<decltype(volume_fluxes)>::tags_list>>
+      div_fluxes{volume_mesh.number_of_grid_points()};
+  divergence(make_not_null(&div_fluxes), volume_fluxes, volume_mesh,
+             volume_inv_jacobian);
+  Variables<db::wrap_tags_in<Tags::dt, tags>> expected_dt_vars{
+      volume_mesh.number_of_grid_points()};
+  std::copy(div_fluxes.data(), div_fluxes.data() + div_fluxes.size(),
+            expected_dt_vars.data());
+  expected_dt_vars *= -1.0;
```

Comment:
Do I correctly understand that this is the test we discussed in a previous PR, where you are checking "strong_div == weak_div + flux_lift"? If so, it would be good state it either around here to explain how you are constructing the expected vars, or around line 250 to describe the test case as a whole.